### PR TITLE
Today's minor patchs

### PR DESCRIPTION
**Pull request by jason-hwang**
- Added new favicon for multi resolution devices.
- Refactoring view files ( Divide header file )
- Added console log for developer.

### CODE REVIEW

File: lib/scheduler.js [hunk 1]
```diff
@@ -140,19 +140,17 @@ Scheduler.prototype.startServer = function(successCallback, failureCallback) {
       var host = server.address().address;
       var port = server.address().port;
 
-      debug('secc-scheduler listening at http://%s:%s', host, port);
+      console.log('secc-scheduler listening at http://%s:%s', host, port);
```
Comment (ivere27) [inline lib/scheduler.js:143]: even it's a minor patch, better to separate this to another patch. 

one for 'console.log' patch
one for 'favicon' patch, etc.

File: views/header.ejs [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<link rel="apple-touch-icon" sizes="57x57" href="/favicons/apple-icon-57x57.png">
+<link rel="apple-touch-icon" sizes="60x60" href="/favicons/apple-icon-60x60.png">
+<link rel="apple-touch-icon" sizes="72x72" href="/favicons/apple-icon-72x72.png">
+<link rel="apple-touch-icon" sizes="76x76" href="/favicons/apple-icon-76x76.png">
+<link rel="apple-touch-icon" sizes="114x114" href="/favicons/apple-icon-114x114.png">
+<link rel="apple-touch-icon" sizes="120x120" href="/favicons/apple-icon-120x120.png">
+<link rel="apple-touch-icon" sizes="144x144" href="/favicons/apple-icon-144x144.png">
+<link rel="apple-touch-icon" sizes="152x152" href="/favicons/apple-icon-152x152.png">
+<link rel="apple-touch-icon" sizes="180x180" href="/favicons/apple-icon-180x180.png">
```
Comment (ivere27) [inline views/header.ejs:9]: could we just use one icon file(favicon.png for all) ?

File: public/favicons/browserconfig.xml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig><msapplication><tile><square70x70logo src="/ms-icon-70x70.png"/><square150x150logo src="/ms-icon-150x150.png"/><square310x310logo src="/ms-icon-310x310.png"/><TileColor>#ffffff</TileColor></tile></msapplication></browserconfig>
```
Comment (ivere27) [inline public/favicons/browserconfig.xml:2]: this is necessary file for all browsers?

File: public/favicons/manifest.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
+{
```
Comment (ivere27) [inline public/favicons/manifest.json:1]: is this file necessary for all browsers also?
manifest.json is  for responsive web, isn't it?

**Comment by jassonbla** [comment]
I agree your comment. 
I will pull request about fixed & seperate patchs.
